sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -250,6 +251,28 @@ def test_one_hot_encoder_handle_unknown():
     assert_raises(ValueError, oh.fit, X)
 
 
+def test_one_hot_encoder_not_fitted():
+    X = np.array([['a'], ['b']])
+    enc = OneHotEncoder(categories=['a', 'b'])
+    msg = ("This OneHotEncoder instance is not fitted yet. "
+           "Call 'fit' with appropriate arguments before using this method.")
+    with pytest.raises(NotFittedError, match=msg):
+        enc.transform(X)
+
+
+def test_one_hot_encoder_no_categorical_features():
+    X = np.array([[3, 2, 1], [0, 1, 1]], dtype='float64')
+
+    cat = [False, False, False]
+    enc = OneHotEncoder(categorical_features=cat)
+    with ignore_warnings(category=(DeprecationWarning, FutureWarning)):
+        X_tr = enc.fit_transform(X)
+    expected_features = np.array(list(), dtype='object')
+    assert_array_equal(X, X_tr)
+    assert_equal(enc.categories_, list())
```

Comment:
With the adoption of pytest, we are phasing out use of test helpers `assert_equal`, `assert_true`, etc. Please use bare `assert` statements, e.g. `assert x == y`, `assert not x`, etc.

Reply:
Is this true for `assert_array_equal`? Is it preferable to do `assert np.array_equal(x, y)`